docs/java-rest/high-level/apis/main.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[java-rest-high-main]]
+=== Info API
+
+[[java-rest-high-main-request]]
+==== Execution
+
+Cluster information can be retrieved using the `info()` method:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/CRUDDocumentationIT.java[main-execute]
```

Comment:
It isn't really a Create, Read, Update, or Delete operation so maybe it should be in its own test just for naming's sake.